x-pack/plugins/siem/public/resolver/view/submenu.tsx

```diff
@@ -123,6 +134,8 @@ const NodeSubMenuComponents = React.memo(
       [menuAction]
     );
 
+    const closePopover = () => setMenuOpen(false);
```

Comment:
I think you should use `useCallback` here

Reply:
I can throw it in one, but it may be counter productive since it really doesn't have any dependencies. There was this article I read a little while ago from Kent Dodds that goes into it a bit: https://kentcdodds.com/blog/usememo-and-usecallback

Reply:
> Also re: the article. While I agree w/ a lot of what is said there. The author suggests not to use `React.memo`. I would argue the opposite. In fact I'd argue that React shouldn't have any other way to create components.

Completely agree with this